### ISSUE TITLE
Multisig unnecessarily uses bonded aggregates & UI bug

### ISSUE THREAD

**Issue by kodty** [symbol/symbol-desktop-wallet#1344]
Reproduce bug: 
- Create multisig 1 of 1
- Send multisig transfer uisng cosigner with <10 xem

Result:
- Transaction successfully signed pop up appears, but transaction does not broadcast.

Usability: There is no need (afaik) to use bonded multisig, the wallet should be able to detect that the required signers are available. If aggregate complete is used, there is no need for the cosigner to have XYM if 0 fee transaction is possible.

**Comment by jabo38** [comment]
I would add, it isn't just 1-of-1, it is any 1-of-n multisig tx. Also, doing it as a complete creates less blockchain bloat while being faster.

**Comment by cryptoBeliever** [comment]
As @kodty and @jabo38 write above. If only one signature needed wallet should announce Aggregate Completed instead of Aggregate Bonded.

**Comment by cryptoBeliever** [comment]
@AnthonyLaw 
After change I'm not able create multisig. Video: https://share.getcloudapp.com/YEuZxAXR
I got error:

FormTransactionBase.ts?eb0f:292 Uncaught (in promise) TypeError: Cannot read property 'minApproval' of undefined
    at FormMultisigAccountModificationTransaction.getTransactionCommandMode (FormTransactionBase.ts?eb0f:292)
    at FormMultisigAccountModificationTransaction.createTransactionCommand (FormMultisigAccountModificationTransactionTs.ts?c467:403)
    at FormMultisigAccountModificationTransaction.onSubmit (FormTransactionBase.ts?eb0f:333)
    at VueComponent.eval (vee-validate.esm.js?7bb1:1882)
    at step (vee-validate.esm.js?7bb1:66)
    at Object.eval [as next] (vee-validate.esm.js?7bb1:47)
    at fulfilled (vee-validate.esm.js?7bb1:37)

**Comment by AnthonyLaw** [comment]
> @AnthonyLaw
> After change I'm not able create multisig. Video: https://share.getcloudapp.com/YEuZxAXR
> I got error:
> 
> FormTransactionBase.ts?eb0f:292 Uncaught (in promise) TypeError: Cannot read property 'minApproval' of undefined
> at FormMultisigAccountModificationTransaction.getTransactionCommandMode (FormTransactionBase.ts?eb0f:292)
> at FormMultisigAccountModificationTransaction.createTransactionCommand (FormMultisigAccountModificationTransactionTs.ts?c467:403)
> at FormMultisigAccountModificationTransaction.onSubmit (FormTransactionBase.ts?eb0f:333)
> at VueComponent.eval (vee-validate.esm.js?7bb1:1882)
> at step (vee-validate.esm.js?7bb1:66)
> at Object.eval [as next] (vee-validate.esm.js?7bb1:47)
> at fulfilled (vee-validate.esm.js?7bb1:37)

Thanks for the fix back, i will take a look on it :)

**Comment by cryptoBeliever** [comment]
Fixed in https://github.com/symbol/symbol-desktop-wallet/pull/1619

**Comment by cryptoBeliever** [comment]
@AnthonyLaw I've found one more issue regarding this one.
If you create multisig with 2 cosigners and setup 1 - minApproval, 2 minRemoval then wallet is trying to send Aggregate Complate when removing one cosigner but this is invalid.

**Comment by AnthonyLaw** [comment]
hmm thanks for the feedback, I need to check again with those conditions.

**Comment by bassemmagdy** [comment]
@AnthonyLaw should we close that one?

**Comment by cryptoBeliever** [comment]
@bassemmagdy no. It has to be fixed https://github.com/symbol/desktop-wallet/issues/1344#issuecomment-886964195

**Comment by AnthonyLaw** [comment]
Yes, we can't close this.
@AdriaCarrera any update for this?

**Comment by coiki** [comment]
@AnthonyLaw @bassemmagdy @cryptoBeliever moving this back to “In progress” since @AdriaCarrera is currently busy with other things. We’ll need to re-assign this task.

**Comment by cryptoBeliever** [comment]
@coiki @surekabpm 
This issue has to be fixed before next release. If we want release without this fix then this PR's have to be reverted:
- https://github.com/symbol/desktop-wallet/pull/1586
- https://github.com/symbol/desktop-wallet/pull/1619
